src/main/kotlin/no/nav/syfo/util/DateUtil.kt

```diff
@@ -5,3 +5,5 @@ import java.time.*
 val defaultZoneOffset: ZoneOffset = ZoneOffset.UTC
 
 fun nowUTC(): OffsetDateTime = OffsetDateTime.now(defaultZoneOffset)
+
+fun LocalDateTime.toOffsetDateTimeUTC(): OffsetDateTime = this.atZone(ZoneId.of("Europe/Oslo")).withZoneSameInstant(ZoneOffset.UTC).toOffsetDateTime()
```

Comment:
Kanskje vi kunne ha en egen 
`val osloTimeZone = ZoneId.of("Europe/Oslo")`

sånn at det blir 

`fun LocalDateTime.toOffsetDateTimeUTC(): OffsetDateTime = this.atZone(osloTimeZone).withZoneSameInstant(defaultZoneOffset).toOffsetDateTime()`